templates/Chatter/CardChatter.html

```diff
@@ -0,0 +1,43 @@
+<div class="coveo-result-frame" style="padding: 20px; font-size: .9em">
+  <div class="coveo-result-row">
+    <div class="coveo-result-cell" style="width: 32px">
+      <div class="CoveoIcon" data-small="true" data-with-label="false">
+      </div>
+    </div>
+    <div class="coveo-result-cell" style="text-align:left; font-size:1.3em; padding-left:1em">
```

Comment:
em values are not supported by the interface editor : You want to use px values, for every templates.